services/motion/motion.go

```diff
@@ -79,6 +79,7 @@ type MoveOnMapReq struct {
 	Destination   spatialmath.Pose
 	SlamName      resource.Name
 	MotionCfg     *MotionConfiguration
+	Obstacles     []spatialmath.Geometry
```

Comment:
Please also update the `func (r MoveOnMapReq) String() string {` to also print obstacles